addons/l10n_no/models/res_partner.py

```diff
@@ -0,0 +1,8 @@
+# coding: utf-8
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_no_bronnoysund_number = fields.Char(string='Bronnoysund company registry', size=9)
```

Comment:
It's missing the "ø"s. Also the "company registry" seems to be specifically called "Register of Business Enterprises", might be a good idea to use that as the Brønnøysund Register Centre seems to have a *lot* of registers which I assume are mostly not suitable?

Might also be a good idea to use the full name (Brønnøysundregistrene), as Brønnøysund is technically the name of the town where the center is located.

Reply:
I now mention the register of legal entities, and the full name (translated to English) to avoid confusion.

The reference I used is : https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/ (see 0192).